js/app.js

```diff
@@ -0,0 +1,146 @@
+
+
+//questions
+
+
+var questions = [{
+        title: 'Do ye like yer drinks strong?',
+        answers: ['aye', 'nay'],
+        ingredient: 'strong'
+    }, {
+        title: 'Do ye like it with a salty tang?',
+        answers: ['aye', 'nay'],
+        ingredient: 'salty'
+    }, {
+        title: 'Are ye a lubber who likes it bitter?',
+        answers: ['aye', 'nay'],
+        ingredient: 'bitter'
+
+    }, {
+        title: 'Would ye like a bit of sweetness with yer poison?',
+        answers: ['aye', 'nay'],
+        ingredient:'sweet'
+    }, {
+        title: 'Are ye one for a fruity finish?',
+        answers: ['aye', 'nay'],
+        ingredient: 'fruity'
+    },
+
+];
+
+
+console.log(questions);
+
+//ingredients
+
+var pantry = {
+
+    strong:['glug of rum', 'slug of whiskey', 'splash of gin'],
+    salty: ['olive on a stick', 'salt-dusted rim', 'rasher of bacon'],
+    bitter:['shake of bitters', 'spalsh of tonic', 'twist of lemon peel'],
+    sweet:['sugar cube', 'spoonful of honey', 'splash of cola'],
+    fruity:['slice of orange', 'dash of cassis', 'cherry on top']
+    
+};
+
+//constructor for chosen ingredients
+
+var Order = function(){
+	this.order=[]	
+};
+
+Order.prototype.addIngredient=function(category){
+	var ingredients = pantry[category]
+	var randomId = Math.floor(Math.random() * ingredients.length)
+	this.order.push(ingredients[randomId])
+}
+Order.prototype.getFinalOrder=function(){
+	if (this.order.length === 0){
+		return 'You didn\'t pick anything!'
+	}
+	return this.order.join(', ')
+}
+var order = new Order 
```

Comment:
Order is a function, so you'll want to invoke it here: `var order = new Order()`